src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -149,8 +150,8 @@ export class StudySummaryTab extends React.Component<IStudySummaryTabProps, {}>
                     props.onResetSelection = this.handlers.onValueSelection;
                 }
                 props.onChangeChartType = this.handlers.onChangeChartType;
-                props.getData = () => this.store.isCustomChart(chartMeta.uniqueKey) ? this.store.getCustomChartDownloadData(chartMeta) : this.store.getClinicalData(chartMeta);
-                props.downloadTypes = ["Data", "SVG", "PDF"];
+                props.getData = (dataType?: DataType) => this.store.getPieChartDataDownload(chartMeta, dataType);
+                props.downloadTypes = this.store.isCustomChart(chartMeta.uniqueKey) ? ["Summary Data", "SVG", "PDF"] : ["Summary Data", "Full Data", "SVG", "PDF"];
```

Comment:
maybe a commend explaining why we don't do full data for custom?

Reply:
this comment might belong somewhere else.  but would be nice to have somewhere

Reply:
I agree. Actually after second thought, the custom chart should support  both downloads. Just updated the pr.